### ISSUE TITLE
override automatic capitalization on the Resource & Contribution pages

### ISSUE THREAD

**Issue by xinemata** [Kcaden20/critical-code-cookbook#16]
Would it be possible to override kirby's automatic capitalization on the Resource & Contribution pages?

For instance: alejandro t. acierto spells his name in all lower cap, and codexdesign is also meant to be in all lower cap. 

<img width="1096" alt="Screen Shot 2022-06-22 at 1 31 16 PM" src="https://user-images.githubusercontent.com/9159424/175120839-3d763311-0d33-40fe-8f06-3ddec4c41b8b.png">

**Comment by Kcaden20** [comment]
This was styling on my end actually. I can just remove the CSS that
capitalizes names.

**Comment by xinemata** [comment]
That sounds good to me. Thank you.

**Comment by Kcaden20** [comment]
I've removed autocapitalizing from all styling. I also added in an author field to for resources to distinguish who made it vs any description you want to add. 

If either is left empty, the HTML won't show up at all. 
![Screen Shot 2022-06-23 at 2 50 01 PM](https://user-images.githubusercontent.com/13590895/175375558-cb72b5b0-9835-41c8-90e3-5cb3da7c8bec.png)
![Screen Shot 2022-06-23 at 2 52 15 PM](https://user-images.githubusercontent.com/13590895/175375639-8208965f-1ac3-400d-8baf-91008e0be231.png)
